packages/client/hmi-client/src/services/concept.ts

```diff
@@ -51,6 +52,25 @@ async function getCuriesEntities(curies: Array<string>): Promise<Array<DKG> | nu
 	}
 }
 
+async function searchCuriesEntities(query: string): Promise<Array<DKG>> {
+	try {
+		const response = await API.get('/mira/search', {
+			params: {
+				q: query
+			}
+		});
+		if (response?.status !== 200) return [];
+		return response?.data ?? [];
+	} catch (error) {
+		logger.error(`There was an issue searching for mira query: ${query}`, { showToast: false });
+		return [];
+	}
+}
+
+function getCurieUrl(curie: string): string {
+	return `http://34.230.33.149:8772/${curie}`;
+}
```

Comment:
Is this URL actually valid? It looks a little sus from an API routing point of view ... do we have a working example of a curie we can lookup?

Reply:
yes it is valid try http://34.230.33.149:8772/ido:00051
this has been in the code for 18 months now.